libcontainer/configs/config.go

```diff
@@ -33,6 +33,7 @@ type IDMap struct {
 type Seccomp struct {
 	DefaultAction    Action     `json:"default_action"`
 	Architectures    []string   `json:"architectures"`
+	Flags            []string   `json:"flags"`
```

Comment:
So, this is our internal representation. Not sure why we use strings here at all. Could be
```go
const SecccompFilterFlags (
	SeccompTSync = iota
	SeccompFlagLog
	SeccompFlagSpecAllow
)
```
and then specconv can do the conversion.

Yet better, do that in libseccomp-golang can have these. I'll take a look.

Reply:
Did you have a chance to look at libseccomp-golang?